config/mappings/assessment_mapping_fund_round.py

```diff
@@ -137,6 +137,7 @@
                 "Score Subcriteria",
                 "Score",
                 "Score Date Created",
+                "Score Time Created",
```

Comment:
@adamdavies1 Just double checking, it's not possible to rename these columns to just Score Date and Score Time? I don't think we need the word "Created" anymore now we split those up